test/lib/usd/translators/UsdImportMaterialX/UsdImportMaterialX.usda

```diff
@@ -0,0 +1,178 @@
+        interpolation = "faceVarying"
+    )
+    int[] primvars:map1:indices = [0, 92, 14, 92, 93, 18, 92, 18, 14, 14, 18, 15, 93, 94, 16, 93, 16, 18, 18, 16, 17, 18, 17, 15, 15, 17, 1, 0, 95, 92, 19, 20, 23, 19, 23, 12, 12, 23, 13, 20, 3, 21, 20, 21, 23, 23, 21, 22, 23, 22, 13, 13, 22, 2, 0, 96, 95, 24, 25, 28, 24, 28, 19, 19, 28, 20, 25, 4, 26, 25, 26, 28, 28, 26, 27, 28, 27, 20, 20, 27, 3, 0, 29, 96, 29, 30, 33, 29, 33, 96, 96, 33, 97, 30, 5, 31, 30, 31, 33, 33, 31, 32, 33, 32, 97, 97, 32, 98, 0, 14, 29, 14, 15, 36, 14, 36, 29, 29, 36, 30, 15, 1, 34, 15, 34, 36, 36, 34, 35, 36, 35, 30, 30, 35, 5, 1, 17, 37, 17, 16, 41, 17, 41, 37, 37, 41, 38, 16, 94, 99, 16, 99, 41, 41, 99, 40, 41, 40, 38, 38, 40, 7, 2, 22, 42, 22, 21, 46, 22, 46, 42, 42, 46, 43, 21, 3, 44, 21, 44, 46, 46, 44, 45, 46, 45, 43, 43, 45, 8, 3, 27, 47, 27, 26, 51, 27, 51, 47, 47, 51, 48, 26, 4, 49, 26, 49, 51, 51, 49, 50, 51, 50, 48, 48, 50, 9, 98, 32, 100, 32, 31, 56, 32, 56, 100, 100, 56, 53, 31, 5, 54, 31, 54, 56, 56, 54, 55, 56, 55, 53, 53, 55, 10, 5, 35, 57, 35, 34, 61, 35, 61, 57, 57, 61, 58, 34, 1, 59, 34, 59, 61, 61, 59, 60, 61, 60, 58, 58, 60, 6, 1, 37, 59, 37, 38, 64, 37, 64, 59, 59, 64, 60, 38, 7, 62, 38, 62, 64, 64, 62, 63, 64, 63, 60, 60, 63, 6, 2, 42, 39, 42, 43, 67, 42, 67, 39, 39, 67, 40, 43, 8, 65, 43, 65, 67, 67, 65, 66, 67, 66, 40, 40, 66, 7, 3, 47, 44, 47, 48, 70, 47, 70, 44, 44, 70, 45, 48, 9, 68, 48, 68, 70, 70, 68, 69, 70, 69, 45, 45, 69, 8, 4, 52, 49, 52, 53, 73, 52, 73, 49, 49, 73, 50, 53, 10, 71, 53, 71, 73, 73, 71, 72, 73, 72, 50, 50, 72, 9, 5, 57, 54, 57, 58, 76, 57, 76, 54, 54, 76, 55, 58, 6, 74, 58, 74, 76, 76, 74, 75, 76, 75, 55, 55, 75, 10, 6, 63, 77, 63, 62, 81, 63, 81, 77, 77, 81, 78, 62, 7, 79, 62, 79, 81, 81, 79, 80, 81, 80, 78, 78, 80, 11, 7, 66, 79, 66, 65, 84, 66, 84, 79, 79, 84, 80, 65, 8, 82, 65, 82, 84, 84, 82, 83, 84, 83, 80, 80, 83, 11, 8, 69, 82, 69, 68, 87, 69, 87, 82, 82, 87, 83, 68, 9, 85, 68, 85, 87, 87, 85, 86, 87, 86, 83, 83, 86, 11, 9, 72, 85, 72, 71, 90, 72, 90, 85, 85, 90, 86, 71, 10, 88, 71, 88, 90, 90, 88, 89, 90, 89, 86, 86, 89, 11, 10, 75, 88, 75, 74, 91, 75, 91, 88, 88, 91, 89, 74, 6, 77, 74, 77, 91, 91, 77, 78, 91, 78, 89, 89, 78, 11]
+}
+
+def Mesh "pPlatonic2" (
+    references = </pPlatonic1>
+)
+{
+    rel material:binding = </MaterialX/Materials/Jade>
+    double3 xformOp:translate = (-8, 0, 0)
+    uniform token[] xformOpOrder = ["xformOp:translate"]
+}
+
+def Mesh "pPlatonic3" (
+    references = </pPlatonic1>
+)
+{
+    rel material:binding = </MaterialX/Materials/Gold>
+    double3 xformOp:translate = (8, 0, 0)
+    uniform token[] xformOpOrder = ["xformOp:translate"]
+}
+
+over "MaterialX"
+(
+    references = [@./standard_surface_jade.usda@</MaterialX>,
+                  @./standard_surface_gold.mtlx@</MaterialX>]
```

Comment:
Will exercise the SdfFileReference auto-conversion to USD, but requires the UsdMtlx plugin which is not in the USD we currently use for tests.